### ISSUE TITLE
Network profile is saved as wrong filename

### ISSUE THREAD

**Issue by Spitz3492** [friedrith/node-wifi#78]
https://github.com/friedrith/node-wifi/blob/eeaef529e95554e3abea70f2224d2aaece80bb20/src/windows-connect.js#L34

https://github.com/friedrith/node-wifi/blob/eeaef529e95554e3abea70f2224d2aaece80bb20/src/windows-connect.js#L52

The windows connect function saves the new network profile under the filename "nodeWifiConnect.xml" but then immediately after uses the netsh command to add the profile using the filename (wifi ssid + ".xml"). This gives me the error, "stdout: 'The system cannot find the file specified.\r\n\r\n\r\n',". So line 34 should write the filename as (ap.ssid + ".xml") not "nodeWifiConnect.xml". After I made this change to the writeFileAsync and the following execCommand delete on line 46, my program connecting to the network successfully.

**Comment by samuelcseto** [comment]
Hi, I have already fixed this in #76. The library later tries to delete the file named 'nodeWifiConnect.xml' So I guess the correct file name should be 'nodeWifiConnect.xml' and not the SSID.

**Comment by SalatielSauer** [comment]
Hello, I believe the problem still persists in the [NPM version](https://www.npmjs.com/package/node-wifi).

**Comment by samuelcseto** [comment]
The pull request is still open so that's why.

S pozdravom,
Samuel Csető


ut 6. 8. 2019 o 15:29 Salatiel <notifications@github.com> napísal(a):

**Comment by Spitz3492** [comment]
Change the filename to "nodeWifiConnect.xml" then as well on [line #40 ](https://github.com/friedrith/node-wifi/blob/eeaef529e95554e3abea70f2224d2aaece80bb20/src/windows-connect.js#L40) because that is what is causing my issue. Your pull is screwing up the master branch of this npm install.

**Comment by Spitz3492** [comment]
I checked out your branch and it seems you have already fixed this issue. Please commit that change to master.

**Comment by samuelcseto** [comment]
I am not the author of this repo so I can't merge into master until the author accepts my pull request #76

**Comment by friedrith** [comment]
Sorry guys, I was in vacation without internet connection. I will merge this pull request #76 as soon as possible.

**Comment by friedrith** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
